### PR TITLE
Add activities to the model

### PR DESCRIPTION
**Pull request by Mathieu414**

### CODE REVIEW

File: database/models/events.db.php [hunk 1]
```diff
@@ -263,7 +267,7 @@ static function fromEventList(array $events)
                 $event['end_date'],
                 $event['deadline'],
                 $event['open'],
-                isset($event['present']) ? $event['present'] : null
+                isset ($event['present']) ? $event['present'] : null
```
Comment (Atmos4) [inline database/models/events.db.php:270]: This change is a bit odd. I am not familiar enough with PHP logic to know if it breaks anything

File: database/models/users.db.php [hunk 2]
```diff
@@ -132,7 +136,7 @@ static function getCurrent(): User|null
         if (!has_session("user_id")) {
             return null;
         }
-        if (isset($_SESSION['controlled_user_id'])) {
+        if (isset ($_SESSION['controlled_user_id'])) {
```
Comment (Atmos4) [inline database/models/users.db.php:139]: same here. What formatter do you use?

**Review by Atmos4 — DISMISSED** [review]
Should be good to merge regardless, but I would appreciate it if you would test and see if the syntax changes I commented about work well!

File: database/models/users.db.php (hunk 2 above)
Comment (Mathieu414) [reply database/models/users.db.php:139]: I use PSR-12 code formatter, with PHP extension of devsense.
I checked, and `isset` is actually not a normal function, but a language construct that is used like a function. Hence the white space added by the formatter like you would add one for an `if` 🤯

File: database/models/events.db.php (hunk 1 above)
Comment (Mathieu414) [reply database/models/events.db.php:270]: See above 👇

**Review by Atmos4 — APPROVED** [review]

**Pull request merged**